explanation/CalibratedFPSPhot.ipynb

```diff
@@ -1081,9 +1081,14 @@
     "\n",
     "### Baseline correction\n",
     "\n",
-    "By default, the baseline for each `fcqfid` is determined using observations obtained more than *100 d prior* to the time of maximum light. This threshold is extremely conservative as very few transients have a rise time >100 d. If there are not enough suitable pre-transient observations (i.e., < 25) to measure the baseline correction, the baseline is determined using observations obtained more than *500 d after* the time of maximum light. Again this choice is conservative, though there are transients that have long rise times or that can be detected for years after they are discovered. The baseline correction `C` is calculated via the median `forcediffimflux` value in the baseline region.\n",
+    "By default, the baseline for each `fcqfid` is determined using observations obtained more than *100 d prior* to the time of maximum light and after when transients have faded to 22.5 mag calculated based on $^{56}$Co decay rate. If there are no observations near the time of maximum light for the calculation, observations obtained 611 days after the time of maximum light are used. These thresholds are extremely conservative, though there are transients that have long rise times or that can be detected for years after they are discovered. Warnings are implemented in order to capture such instances. The baseline correction `C` is calculated via the median `forcediffimflux` value in the baseline region, and uncertainty of the baseline `C` is obtained using bootstrapping.\n",
```

Comment:
By default, the baseline for each `fcqfid` is determined using observations obtained more than *100 d prior* to the time of maximum light and long after the transient has faded below the ZTF detection threshold. This time is (conservatively) determined by assuming the peak luminosity of the transient is purely powered by radioactive $^{56}$Co, and we then calculate the time to fade to 22.5 mag in the observed frame assuming the transient is at z = 0.09 (this choice of redshift is conservative).